libethash/ethash.h

```diff
@@ -25,26 +26,17 @@
 #include <stddef.h>
 #include "compiler.h"
 
-#define ETHASH_REVISION 19
-#define ETHASH_DAGSIZE_BYTES_INIT 1073741824U // 2**30
-#define ETHASH_DAG_GROWTH 113000000U
-#define ETHASH_EPOCH_LENGTH 30000U
-#define ETHASH_MIX_BYTES 128
-#define ETHASH_DAG_PARENTS 256
-#define ETHASH_CACHE_ROUNDS 3
-#define ETHASH_ACCESSES 64
-
-// *********************
-// TODO: Remove once other code moved over to compliant naming scheme.
```

Comment:
how rude.

don't just clobber if you don't like. the naming scheme is there for a reason.